lib/core/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": true,
+    "jsx": "preserve",
```

Comment:
oh Vue also use JSX?

Reply:
Yes, it is. Reference: https://v3.vuejs.org/guide/render-function.html#jsx